<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocialChat</class>
 <widget class="QMainWindow" name="SocialChat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1267</width>
    <height>747</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QGroupBox" name="groupBoxBot">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Bot</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QListWidget" name="bots"/>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBoxVisiblePlayers">
        <property name="title">
         <string>Visible players</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="labelVisiblePlayer">
           <property name="text">
            <string>XXXX</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBoxChat">
        <property name="title">
         <string>Chat</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="listViewChatType"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>History and content</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="listViewHistory"/>
           </item>
           <item>
            <widget class="QLineEdit" name="chat">
             <property name="placeholderText">
              <string>Your text</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBoxInformations">
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Bot informations</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="labelImageAvatar">
         <property name="pixmap">
          <pixmap resource="../../client/base/resources/client-resources.qrc">:/images/player_default/front.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelPseudo">
         <property name="text">
          <string>Pseudo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Note:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="note"/>
       </item>
       <item>
        <widget class="QLabel" name="labelClan">
         <property name="text">
          <string>Clan: None</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../../client/base/resources/client-resources.qrc"/>
 </resources>
 <connections/>
</ui>
